ide-common/src/main/kotlin/org/digma/intellij/plugin/editor/FocusedDocumentHelper.kt

```diff
@@ -0,0 +1,22 @@
+package org.digma.intellij.plugin.editor
+
+private var openedFilesList: MutableList<String> = ArrayList()
```

Comment:
Who uses/query the `openedFilesList` ? I see only `add` and `remove`

Reply:
great catch! thanks! I've just remove it. it's no longer needed